CMakeLists.txt

```diff
@@ -65,7 +65,16 @@ if (CMAKE_Fortran_COMPILER_ID STREQUAL "GNU")
     endif()
 elseif ((CMAKE_Fortran_COMPILER_ID STREQUAL "NVHPC") OR (CMAKE_Fortran_COMPILER_ID STREQUAL "PGI"))
     if (CMAKE_Fortran_COMPILER_VERSION VERSION_LESS 21.7)
-        message(FATAL_ERROR "${__err_msg}ERROR: NVHPC v21.7 or newer is required to build MFC.")
+        message(FATAL_ERROR "${__err_msg}ERROR: When using NVHPC, v21.7 or newer is required to build MFC.")
+    endif()
+    if (MAKE_BUILD_TYPE STREQUAL "Debug")
+        if (MFC_OpenACC)
+            if (CMAKE_Fortran_COMPILER_VERSION VERSION_LESS 23.0)
+                message(FATAL_ERROR "${__err_msg}ERROR: When using NVHPC, v23.0 or newer is required to build MFC with Debug and GPU options.")
```

Comment:
I am not sure if this is relevant (or important) but once a fix comes out, if you try to build an older version of NVHPC is Debug mode, you will get this fatal error and would have to remove it manually from the CMakLists.txt. Do you have any thoughts on this?

Reply:
It seems that the logic is there such that you only get this error if NVHPC is out of date and won't work anyway, no? 

Either you have some version 23.5 or so = works and no message; or, an old version 22.5 or so = doesn't work and get this message.

Reply:
I have no idea if they will. If they do, I guess we could go back and change the CMake file. But, even if they patch old versions, those versions are unlikely to be actually installed on clusters anyway. Either they will have the original old versions, or a newer proper major/minor version (or both), but not a patched old version. At least in my experience.

Reply:
Taking another look at nice, I realize I meant this for the case of NVHPC in CPU + Debug mode. I don't think this is too much of a problem since we'll just remove it once it is fixed but you can add a version check if you feel so inclined.

Reply:
added